src/App.jsx

```diff
@@ -14,70 +16,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```
It's a convention to name your state this way:
`[state, setState]`
`[goods, setGoods]`
`[whateverStateNameYouMayHave, setWhateverStateNameYouMayHave]`

You may read more about it and other React naming conventions [here](https://medium.com/@kristiyan.velkov/react-js-naming-convention-fe83acfbafb3).